### PR TITLE
Rename remaining questionSet instances to qAndA

### PR DESCRIPTION
**Pull request by tengjianling**

### CODE REVIEW

**Review by ChengJiyuqing — APPROVED** [review]

File: src/main/java/seedu/medmoriser/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -19,23 +19,23 @@
...
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_ANSWER + "yes "
+            + PREFIX_TAG + "general knowledge "
+            + PREFIX_TAG + "common sense";
 
-    public static final String MESSAGE_SUCCESS = "New questionSet added: %1$s";
-    public static final String MESSAGE_DUPLICATE_QUESTIONSET = "This questionSet already exists in the answer book";
+    public static final String MESSAGE_SUCCESS = "New qAndA added: %1$s";
+    public static final String MESSAGE_DUPLICATE_QANDA = "This qAndA already exists in the answer book";
```
Comment (jonfoocy) [inline src/main/java/seedu/medmoriser/logic/commands/AddCommand.java:38]: Change to `QAndA` for consistency with line 22?

Comment (jonfoocy) [inline src/main/java/seedu/medmoriser/logic/commands/AddCommand.java]: Change to `QAndA` for consistency with line 22?

File: src/main/java/seedu/medmoriser/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -42,23 +42,23 @@
             + PREFIX_QUESTION + "What causes pimples? "
             + PREFIX_ANSWER + "triggered by androgen hormones and, in some cases, genetics";
 
-    public static final String MESSAGE_EDIT_QUESTIONSET_SUCCESS = "Edited questionSet: %1$s";
+    public static final String MESSAGE_EDIT_QANDA_SUCCESS = "Edited qAndA: %1$s";
```
Comment (jonfoocy) [inline src/main/java/seedu/medmoriser/logic/commands/EditCommand.java]: Change to `QAndA` for consistency

File: src/main/java/seedu/medmoriser/logic/commands/FindCommand.java [hunk 3]
```diff
@@ -14,14 +14,14 @@
...
- * Finds and lists all questionSets in address book whose name contains any of the argument keywords.
+ * Finds and lists all qAndAs in address book whose name contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all questionSets whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all qAndAs whose names contain any of "
```
Comment (jonfoocy) [inline src/main/java/seedu/medmoriser/logic/commands/FindCommand.java]: `QAndA`

File: src/main/java/seedu/medmoriser/model/qanda/QAndAContainsKeywordsPredicate.java [hunk 4]
```diff
@@ -6,7 +6,7 @@
 import seedu.medmoriser.commons.util.StringUtil;
 
 /**
- * Tests that a {@code QuestionSet}'s {@code Question or Answer} matches any of the keywords given.
+ * Tests that a {@code QandA}'s {@code Question or Answer} matches any of the keywords given.
```
Comment (jonfoocy) [inline src/main/java/seedu/medmoriser/model/qanda/QAndAContainsKeywordsPredicate.java:9]: `QAndA`

File: src/main/java/seedu/medmoriser/model/qanda/exceptions/DuplicateQAndAException.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package seedu.medmoriser.model.qanda.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate QAndAs (QAndAs are considered
+ * duplicates if they have the same identity).
+ */
+public class DuplicateQAndAException extends RuntimeException {
+    public DuplicateQAndAException() {
+        super("Operation would result in duplicate question sets");
```
Comment (jonfoocy) [inline src/main/java/seedu/medmoriser/model/qanda/exceptions/DuplicateQAndAException.java]: "... duplicate QAndAs"

**Review by jonfoocy — CHANGES_REQUESTED** [review]
Just some changes for consistency then good to go!

**Review by jonfoocy — APPROVED** [review]
Looks great!

**Pull request merged**